src/pages/Validators/ValidatorsTable.tsx

```diff
@@ -378,7 +378,12 @@ function ValidatorRow({validator, columns}: ValidatorRowProps) {
 
   return (
     <GeneralTableRow
-      onClick={inDev ? () => rowClick(validator.owner_address) : undefined}
+      onClick={
+        inDev &&
+        WHILTELISTED_TESTNET_DELEGATION_NODES.includes(validator.owner_address)
+          ? () => rowClick(validator.owner_address)
+          : undefined
+      }
```

Comment:
this is why i think normal nodes and delegatory node should be two tables. there'll be more and more if/else checks. let's make sure we have a BE follow-up to address this.

Reply:
yes! thats what i thought and here's the PR for the separation: https://github.com/aptos-labs/explorer/pull/358